nginx.conf

```diff
@@ -7,6 +7,37 @@ http {
     access_log /dev/stdout;
     listen 8080;
     
+    root /usr/share/nginx/html;
+    index index.html;
+
+    ssi on;
+    ssi_types text/html;
+
+    location / {
+      ssi on;
+      try_files $uri $uri/ =404;
+    }
+
+    location /remote_stage {
+      rewrite ^/remote_stage/(.*)$ /$1 break;
+      proxy_pass https://developers.stage.redhat.com;
+    }
+
+    location /remote {
+      rewrite ^/remote/(.*)$ /$1 break;
+      proxy_pass https://developers.redhat.com;
```

Comment:
Is any of this still required?
It looks like we moved away from nginx loading to loading it from the frontend